content/docs/start/model-management/model-registry.md

```diff
@@ -0,0 +1,297 @@
+  versions and lifecycle stages.'
+---
+
+# Get Started: Model Registry
+
+Just as we use experiment tracking to manage model development, it is a good
+idea to keep a <abbr>model registry</abbr> to manage the lifecycle of the models
+we get from our experiments. Using DVC and DVC [Studio](/doc/studio) we will set
+up a model registry where we can discover, share, deploy and audit all our
+models and which will serve as the single source of truth for our model
+management.
+
+You can also set up the model registry locally without Studio, using a command
+line tool called [Git Tag Ops (GTO)](<(/doc/gto)>). We will see how this is done
+[later in this chapter](#GTO-tip).
+
+<details id="follow-along-instructions">
+
+#### 💡 Expand to see how to set things up to follow along with the guide
+
+You can fork our
+[example repository](https://github.com/iterative/example-get-started-experiments)
+and follow the
+[installation steps](https://github.com/iterative/example-get-started-experiments#installation)
+to set it up locally. To perform the model registry actions in this guide, sign
+up to Studio, connect Studio to your Git Repository and add the forked project
+using
+[this guide](/doc/studio/user-guide/experiments/create-a-project#connect-to-a-git-repository-and-add-a-project).
+
+</details>
+
+## Adding models
+
+Let's now train a model and add it to the model registry. We will be using
+[DVCLive](/doc/dvclive) and add a model using Python code. This will also
+automatically save the model to DVC.
+
+We use the [`log_artifact`](/doc/dvclive/live/log_artifact) method to
+<abbr>cache</abbr> the model with DVC and add it to the the model registry. Open
+the training script `src/train.py` in our example repository and you will see
+the following code under the `with Live(...)` context:
+
+```python
+with Live(...) as live:
+
+...
+
+    live.log_artifact(
+        path="models/model.pkl",
+        type="model",
+        name="pool-segmentation",
+        desc="This is a Computer Vision (CV) model that's segmenting out swimming pools from satellite images.",
+        labels=["cv", "segmentation", "satellite-images"],
+    )
+```
+
+Here the `path` parameter tells DVC that our model is to be found under
+`"models/model.pkl"`, the `type` parameter is `"model"` and so it will show up
+in the Studio registry (other artifact types will not) and the rest of the
+parameters are descriptive and optional and will also show up in the model
+registry.
+
+Now we just need to run the python script which includes this code to cache and
+register the model. If you are following our example repository then the
+training script is included in a DVC pipeline
+[we prepared](/doc/start/experiments/experiment-pipelines) in the Experiment
+Management guide. so we can just call `dvc exp run` to run the entire experiment
+pipeline.
+
+We now commit the result to git (and push it to our git remote) and the new
```

Comment:
When trying this out, `dvc exp run` does nothing because we haven't changed anything since last run, and so I'm unable to git commit because there are no changes, which might confuse users. Also, technically I can already see the model in Studio at this point.